src/content/protocols/aave.md

```diff
@@ -0,0 +1,916 @@
+
+The protocol does currently have limited validation on asset prices provided by Chainlink. This checks include upper caps for stablecoins and LSTs and a sanity check that the price is above 0 for all assets. If the reported price by the price feed was below 0, a fallback oracle would be queried.
+
+Aave has currently no fallback oracle price feeds instantiated. As a consequence if the price was equal to or below 0, user actions on the `Pool` contract that require a price would revert.
+
+The replacement of a stale or untrusted oracle price feed requires a governance vote on permission level 1 (see Exit Window).
+
+The Chainlink oracle system itself is upgradeable without decentralized ownership over those permissions. This dependency thus introduces centralization risk in the Aave V3 protocol.
+
+### Cross-Chain Vote
+
+Votes are currently held on Polygon by using a system called a.DI for cross-chain communication developed by the Aave Community (BGD Labs).
+
+The a.DI does not rely on a single bridge provider to move messages across different networks, but uses a 2/3 threshold, sending the messages in parallel via different bridge providers, reducing the dependency on a single provider.
+
+If the community cannot or does not want to use the a.DI or the other voting networks, then the community still can carry out the vote on Ethereum Mainnet. This would eliminate the cross-chain dependencies for voting.
+
+The cross-chain voting system has low centralisation risk because of its fault tolerant design, while the oracle system is currently unmitigated.
+
+> Autonomy score: High
+
+## Exit Window
+
+Critical permissions, including protocol upgrades, are controlled by the _Aave Governance_ onchain governance system.
+
+Two levels of permissions are separated in the (current) implementation of _Aave Governance_: _Level 2_ defines permissions on the governance system itself and on the `AAVE` token contract, _Level 1_ covers the remaining permissions.
+
+AAVE holders (also stkAAVE, aAAVE) are able to create new proposals (requires 80,000 / 200,0000 votes for level 1 / level 2) and vote on proposals (at least 320,000 / 1,040,000 votes are required for a valid proposal and a voting differential (yes minus no) of 80,000 / 1,040,000 votes).
+
+Votes start 1 day after proposal creation and the voting period is 10 days for _Level 2_, and 3 days for _Level 1_ proposals.
+
+Furthermore, the execution of successful votes is blocked by an _Exit Window_ with a delay of 7 days for _Level 2_ and 1 day for _Level 1_ permissions.
+
+Hence, the enforced _Exit Windows_ do not fulfill the 30-days delay required for a _Low_ risk assessment and only the _Level 2_ delay of 7 days qualifies for a _Medium_ risk according to our criteria.
+
+However, this is mitigated by the _Aave Governance_ system through the _Aave Governance V3 Guardian_ multisig account, which satisfies the Security Council requirements, and its ability to block unintended proposals from execution.
```

Comment:
Shorten the last three paragraphs to: 

"Furthermore, the execution of successful votes is blocked by an _Exit Window_ with a delay of 7 days for _Level 2_ and 1 day for _Level 1_ permissions. While these _Exit Windows_ do not qualify for a _Low_ or _Medium_ score, the _Aave Governance V3 Guardian_ multisig account (adhering to the _Security Council_ requirements) and its ability to block unintended proposals from execution mitigates this risk."

and add a paragraph:

However, these safeguards do not apply to the various critical permissions controlled by a number of non-_Security Council_ compliant multisig accounts resulting in a _High_ centralization risk score.